### PR TITLE
fix: upgrade tama and remove patch-package

### PR DESCRIPTION
**Pull request by peterjskaltsis**
Looks like we might not need the patch on latest tamagui?

Seems to be functioning fine for me without it, what error were you seeing?

@G-Ray

### CODE REVIEW

File: app/index.tsx [hunk 1]
```diff
@@ -6,32 +6,27 @@ import { useFonts } from "expo-font";
 import config from "../tamagui.config";
 
 export default function App() {
-	const colorScheme = useColorScheme();
+  const colorScheme = useColorScheme();
```
Comment (G-Ray) [inline app/index.tsx]: Please do not reformat files in this PR, open another PR if you want to suggest another formatting rules.

File: package-lock.json
Comment (G-Ray) [inline package-lock.json:1]: I used and listed commands to use with `npm` in the README, please do not remote the package-lock

File: tamagui.config.ts [hunk 2]
```diff
@@ -1,70 +1,69 @@
...
-		damping: 20,
-		stiffness: 60,
-	},
-	quick: {
-		type: "spring",
-		damping: 20,
-		mass: 1.2,
-		stiffness: 250,
-	},
+  bouncy: {
```
Comment (G-Ray) [inline tamagui.config.ts]: Please no reformatting here too

File: yarn.lock
Comment (G-Ray) [inline yarn.lock:1]: We should keep and update `package-lock.json`, and remove this file.

File: package.json [hunk 3]
```diff
@@ -5,13 +5,12 @@
     "android": "TAMAGUI_TARGET=native expo start --android",
     "ios": "TAMAGUI_TARGET=native expo start --ios",
     "web": "TAMAGUI_TARGET=web expo start --web",
-    "postinstall": "patch-package"
+    "upgrade:tamagui": "manypkg upgrade tamagui && manypkg upgrade @tamagui && manypkg upgrade tamagui-loader"
```
Comment (G-Ray) [inline package.json:8]: :+1:

File: tamagui.config.ts [hunk 4]
```diff
@@ -1,70 +1,69 @@
 import { createInterFont } from "@tamagui/font-inter";
 import { createMedia } from "@tamagui/react-native-media-driver";
 import { shorthands } from "@tamagui/shorthands";
-import { themes, tokens } from "@tamagui/theme-base";
+import { themes, tokens } from "@tamagui/themes";
```
Comment (G-Ray) [inline tamagui.config.ts:4]: :+1:

**Comment by G-Ray** [comment]
Thank you @peterjskaltsis !
I requested a few changes to keep the related modifications focused on upgrading tamagui.

Yes the tamagui patch should not be needed anymore. The error was `import` or `import.meta` was undefined in browser.

File: app/index.tsx (hunk 1 above)
Comment (peterjskaltsis) [reply app/index.tsx]: Sorry about that, looks like it auto-formatted on save due to a lack of a prettier config setup.

Are you using tabs?

File: package-lock.json
Comment (peterjskaltsis) [reply package-lock.json:1]: No problem, will update

File: app/index.tsx (hunk 1 above)
Comment (peterjskaltsis) [reply app/index.tsx]: Resolved

**Comment by peterjskaltsis** [comment]
> Thank you @peterjskaltsis ! I requested a few changes to keep the related modifications focused on upgrading tamagui.
> 
> Yes the tamagui patch should not be needed anymore. The error was `import` or `import.meta` was undefined in browser.

Resolved all of your comments, thank you for taking a look and also putting this repo together originally @G-Ray  🙏

**Pull request merged**